reactive/etcd.py

```diff
@@ -308,7 +309,6 @@ def snap_install():
     snap.install('core')
     if channel:
         snap.install('etcd', channel=channel, classic=False)
-    remove_state('etcd.pillowmints')
```

Comment:
Why did you remove this?

Reply:
I have replaced `etcd.pillowmints` with `etcd.ssl.exported` as that's all it seemed to do anymore.

Reply:
Right, but this wasn't replaced with `remove_state('etcd.ssl.exported')`, it was just removed. If we don't clear the `etcd.ssl.exported` flag here, then `render_default_user_ssl_exports` won't be re-run after an upgrade from etcd2 -> etcd3 will it?

Reply:
I think it'll be cleared by L148-149 `@hook('upgrade-charm')` - `def remove_states():`.  I thought this route had to be taken if you were going deb->snap?

Reply:
Nah, I'm talking about snap->snap upgrade from etcd 2.3 to 3.x, where the user does this:

```
juju config etcd channel=3.0/stable
```

It'll cause `snap_install` to run again, but with no `upgrade-charm` event.

Reply:
Ah yeah, I see what you mean.  Fixed!  Thanks.